.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "restructuredtext.confPath": "${workspaceFolder}/docs"
+}
```

Comment:
The VSCode and Sphinx output has been added to the PR.

Reply:
Fixed